File: jscomp/test/gpr_1060_test.ml

type u 

external a : u = "react" [@@bs.module]

external b : unit -> int = "bool" [@@bs.module "react"]

let v = a
let h = b ()

type bla


external foo : bla = "foo.react" [@@bs.module]

external bar : unit -> bla  = "bar" [@@bs.val] [@@bs.module "foo.react"]

let c = foo 

let d = bar ()

File: jscomp/test/gpr_1060_test.js
'use strict';

var Foo = require("foo.react");
var A   = require("react");

var v = A;

var h = A.bool();

var c = Foo;

var d = Foo.bar();

exports.v = v;
exports.h = h;
exports.c = c;
exports.d = d;
/* v Not a pure module */
